saleor/shipping/models.py

```diff
@@ -173,6 +180,17 @@ def __repr__(self):
         )
 
 
+class ShippingMethodZipCode(models.Model):
+    shipping_method = models.ForeignKey(
+        ShippingMethod, on_delete=models.PROTECT, related_name="zip_codes"
```

Comment:
Why `PROTECT` here? I think we can remove zip codes when removing a shipping method?